src/foam/nanos/notification/NotificationView.js

```diff
@@ -3,11 +3,39 @@ foam.CLASS({
   name: 'NotificationView',
   extends: 'foam.u2.View',
   axioms: [ foam.pattern.Faceted.create() ],
+
+  css: `
+   .body {
```

Comment:
Shouldn't this be scoped with ^? Is 'body' a good name, given this already has a specific meaning in HTML.